common/database.cpp

```diff
@@ -2057,12 +2057,18 @@ void Database::PurgeCharacterParcels()
 	pel.event_type_name = PlayerEvent::EventName[pel.event_type_id];
 	std::stringstream ss;
 	for (auto const   &r: results) {
-		pd.from_name = r.from_name;
-		pd.item_id   = r.item_id;
-		pd.note      = r.note;
-		pd.quantity  = r.quantity;
-		pd.sent_date = r.sent_date;
-		pd.char_id   = r.char_id;
+		pd.from_name  = r.from_name;
+		pd.item_id    = r.item_id;
+		pd.note       = r.note;
+		pd.quantity   = r.quantity;
+		pd.sent_date  = r.sent_date;
+		pd.char_id    = r.char_id;
+		pd.aug_slot_1 = r.aug_slot_1;
```

Comment:
Can we move these slots to be directly after the item ID?

Reply:
Kingly, Do you mean in the db structure?

Reply:
Yes, that way they're right beside the item ID column in the repository and database schema for consistency with tables like `inventory`.